apps/block_scout_web/lib/block_scout_web/channels/address_channel.ex

```diff
@@ -39,7 +39,7 @@ defmodule BlockScoutWeb.AddressChannel do
     with {:ok, casted_address_hash} <- AddressHash.cast(socket.assigns.address_hash),
          {:ok, address = %{fetched_coin_balance: balance}} when not is_nil(balance) <-
            Chain.hash_to_address(casted_address_hash),
-         exchange_rate <- Market.get_exchange_rate("cGLD") || Token.null(),
+         exchange_rate <- Market.get_exchange_rate("CELO") || Market.get_exchange_rate("cGLD") || Token.null(),
```

Comment:
What if we just start using `CELO` everywhere, it seems that the symbol has been updated. We could also just do:
```suggestion
         exchange_rate <- Market.get_exchange_rate(Explorer.coin()) || Token.null(),
```
here and everywhere to read the token symbol from the env var.